cli/cmd/login.go

```diff
@@ -217,7 +216,7 @@ func userDisplayName(claims jwt.MapClaims) string {
 // oauth2Login opens a browser, runs a temporary HTTP server to delegate OAuth2 login flow and
 // returns the JWT token and a refresh token (if supported)
 func oauth2Login(ctx context.Context, port int, oidcSettings *settingspkg.OIDCConfig, oauth2conf *oauth2.Config, provider *oidc.Provider) (string, string) {
-	oauth2conf.RedirectURL = fmt.Sprintf("http://localhost:%d/auth/callback", port)
+	oauth2conf.RedirectURL = fmt.Sprintf("https://localhost:%d/auth/callback", port)
```

Comment:
This should still be http, it is interacting with ArgoCD (not Atlas)

Reply:
Changed to http